linebot/messaging_api/api_messaging_api.go

```diff
@@ -935,9 +935,9 @@ func (client *MessagingApiAPI) GetFollowersWithHttpInfo(
 		return nil, nil, err
 	}
 
-	var query url.Values
-	query = url.Values{"start": []string{start}}
-	query = url.Values{"limit": []string{strconv.FormatInt(int64(limit), 10)}}
+	query := url.Values{}
+	query.Add("start", start)
+	query.Add("limit", strconv.FormatInt(int64(limit), 10))
```

Comment:
(Not related to this change) Hmm... `start` is one of optional parameters, but it requires string value like `""`. Perhaps calling `GetFollowers` would fail in current implementation... This seems bug...

Reply:
@Yang-33 Agree with you. I can help check what will happen if we put empty string as a query-param value. Maybe net/http might just ignore and omit it.

But if it doesn't, might need to change the generator to properly handle optional query-params (maybe if-else to check if it's optional before doing url.Values.Add(...).

Reply:
@Yang-33  Just tested with the following, it seems `net/http` doesn't omit the query-parameter when it's empty string...

```
func TestGetFollowers_ItShouldNotPassStartWhenItIsEmptyString(t *testing.T) {
	server := httptest.NewServer(
		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
			if r.URL.Query().Has("start") {
				w.Header().Set("TEST-ERROR", "incorrect start being sent from client. does not expect to have `start` in query-parameter")
				w.WriteHeader(http.StatusInternalServerError)
				return
			}
			json.NewEncoder(w).Encode(messaging_api.GetFollowersResponse{UserIds: []string{}, Next: "abcdef"})
		}),
	)
	client, err := messaging_api.NewMessagingApiAPI(
		"channelToken",
		messaging_api.WithEndpoint(server.URL),
	)
	if err != nil {
		t.Fatalf("Failed to create client: %v", err)
	}
	resp, _, _ := client.GetFollowersWithHttpInfo("", 1000)
	if resp.StatusCode != http.StatusOK {
		t.Errorf("Not getting 200 response back: %s", resp.Header.Get("TEST-ERROR"))
	}
}
```

Reply:
@Yang-33 Please don't merge this PR yet then. I will figure it out how to deal with optional query-parameter correctly next weekend.

Reply:
Thank you for checking this 🙏 I see, let's keep this PR.

Reply:
There are at least two options.
(1) Use a pointer. However, since line-bot-sdk-go generates code automatically, all functions will use a pointer. (Or, only optional parameters use a pointer.)
(2) As before, if it is `""`, do not specify this to request.

Personally, I like (1), but I don't know whether (1) is preferable as Go language.
I'd like to hear @kkdai 's opinion.

Reply:
Hi @Yang-33 and @mekpavit 
I agree with @Yang-33 -san, the option (1) is much better.
Because using optional parameters might be confusing if only two but both two might be nil. 

```
process(&value1, nil) // Only value1 is provided
process(nil, &value2) // Only value2 is provided
process(nil, nil)     // Neither is provided
process(&value1, &value2) // Both are provided
```